packages/harmony/src/components/input/TextInput/TextInput.module.css

```diff
@@ -0,0 +1,147 @@
+.root {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+}
+
+.inputRoot {
+  display: flex;
+  align-items: center;
+  gap: var(--harmony-unit-3);
+  padding: var(--harmony-unit-4);
+  border: 1px solid var(--harmony-border-default);
+  border-radius: var(--harmony-unit-1);
+  background-color: var(--harmony-bg-surface-1);
+  transition: border ease-in-out 0.1s;
+  box-sizing: border-box;
+  & svg path {
+    fill: var(--harmony-text-subdued);
+  }
+}
+
+.small {
+  --gap: -4px;
+  --font-size: var(--harmony-font-s);
+  height: 34px;
+}
+
+.default {
+  --gap: 0px;
+  --font-size: var(--harmony-font-m);
+  height: 64px;
+}
+
+.large {
+  --gap: var(--harmony-unit-1);
+  --font-size: var(--harmony-font-xl);
+  height: 66px;
+}
+
+.root:hover {
+  border-color: var(--harmony-neutral-light-6);
+}
+
+.focused {
+  border-color: var(--harmony-secondary);
+}
+
+.warning,
+.warning.focused {
+  border-color: var(--harmony-orange);
+}
+
+.error,
+.error.focused {
+  border-color: var(--harmony-red);
+}
+
+.error:hover,
+.error:hover.focused {
+  /* TODO: Figure out of we want a hover color */
+  border-color: var(--harmony-red);
+}
+
+.disabled {
+  background-color: var(--harmony-neutral-light-9);
+  border-color: var(--harmony-neutral-light-8);
+}
+
+.textInput {
+  width: 100%;
+  height: 100%;
+  outline: 0;
+  border: 0;
+  background: none;
+  font-size: var(--harmony-font-size);
```

Comment:
I don't think this exists. We have a variety of sizes like `--harmony-font-xs`, etc.
(leaving one comment but there are other references to this)